aiosmtpd/smtp.py

```diff
@@ -78,13 +97,38 @@ def max_command_size_limit(self):
             return self.command_size_limit
 
     def connection_made(self, transport):
-        super().connection_made(transport)
-        self.peer = transport.get_extra_info('peername')
-        self.transport = transport
-        log.info('Peer: %s', repr(self.peer))
-        # Process the client's requests.
-        self.connection_closed = False
-        self._handler_coroutine = self.loop.create_task(self._handle_client())
+        if(self.transport is not None
+                and isinstance(transport, sslproto._SSLProtocolTransport)):
+            # It is STARTTLS connection over normal connection
+            self._reader._transport = transport
+            self._writer._transport = transport
+            self.transport = transport
+            # Reset state due to rfc3207 part 4.2
+            self._set_rset_state()
+            self.seen_greeting = False
+            # Do SSL certificate checking as rfc3207 part 4.1 says
+            # Why _extra is protected attribute?
+            extra = self._tls_protocol._extra
+            if hasattr(self.event_handler, 'handle_tls_handshake'):
+                self._tls_handshake_failed = \
+                    not self.event_handler.handle_tls_handshake(
+                        extra['ssl_object'],
+                        extra['peercert'],
+                        extra['cipher'],
+                    )
```

Comment:
Project style is to indent the closing parenthesis to under the previous line's first non-whitespace character.  Can you indent this (and following closing parens) 4 more spaces?